docs/en/compute/AWS-EMR.md

```diff
@@ -71,7 +71,7 @@ $ aws emr create-cluster \
 --bootstrap-actions \
 Path=s3://bucket/path/to/alluxio-emr.sh,\
 Args=[<download-url>,<root-ufs-uri>,<optional-path-to-site-properties>,<optional-additional-delimited-properties>,<optional-property-delimiter>] \
---configurations file://${ALLUXIO_HOME}/integration/emr/alluxio-emr.json \
+--configurations https://alluxio-public.s3.amazonaws.com/emr/2.0.1/alluxio-emr.json \
```

Comment:
@ns1123 Wasn't the requirement that it was an s3 URI?

Reply:
the json file is a local file. The bootstrap needs to be an S3 URI. So this section does need to be updated as well

Reply:
According to `aws emr create-cluster help` that states

```
          Specifies a JSON file that contains  configuration  classifications,
          which you can use to customize applications that Amazon EMR installs
          when cluster instances launch. Applies only to Amazon  EMR  4.0  and
          later.  The  file referenced can either be stored locally (for exam-
          ple, --configurations file://configurations.json ) or stored in Ama-
          zon    S3    (for   example,   --configurations   https://s3.amazon-
          aws.com/myBucket/configurations.json ). Each classification  usually
          corresponds  to  the xml configuration file for an application, such
          as yarn-site for YARN. For a list of available configuration classi-
          fications  and  example  JSON, see the following topic in the Amazon
          EMR Release Guide:
```

The http URL is fine here

Reply:
Thanks for confirming!